android/build.gradle

```diff
@@ -31,6 +35,7 @@ repositories {
 }
 
 dependencies {
-    implementation 'com.facebook.react:react-native:+'
-    api 'com.heapanalytics.android:heap-android-client:0.8.3'
+    implementation "com.facebook.react:react-native:${safeExtGet('reactNativeVersion', '+')}"
```

Comment:
Just curious (I am not a gradle guy at all...) - where does `reactNativeVersion` get set?  Is that something that will reliably be there on all installations?

Reply:
To be honest, I've still got a lot to learn about gradle too... 
- reactNativeVersion comes from the dependencies section as far as I can tell.

I came across this when react-native-heap (and a couple other libraries) failed to compile for me. Looking at the build.gradle for some other libraries, all of them used `safeExtGet` for other dependencies. For example [react-native-device-info](https://github.com/react-native-community/react-native-device-info/blob/master/android/build.gradle).

reactNativeVersion won't always be set, but that's ok b/c `safeExtGet` will use the default value if it's not defined.